index.html

```diff
@@ -12,34 +11,38 @@
 <body>
 
   
-  <div class="title bar">
-    <h1>Searchy</h1>
+  <div class="titlebar">
+    <h1>Make your own mailer!</h1>
   </div>
-  <header>
+  <div class="unitelogo">
+
+  <img src="https://raw.githubusercontent.com/morganwk/bernard/master/unite%20dark%20background%20logo.svg">
+
+</div><header>
     <h2 class="Title">WELCOME!</h2>
-    <p>
-      You can use this app to build a custom mailer for your friends around the county - AND, if you want, for all your neighbors! 
-    </p>
-    <p>
-      Once you've entered your information, we'll give you address labels and mailing materials.
-    </p>
+    
+    <p>You can use this app to build <strong>personal mailer</strong> for your friends around Allegheny County.
+
+</p>
     <p>
       You can use this app to support any of the following:
 
-      <ul>- The Slate of Eight</ul>
+      </p><ul>- The <a href="https://www.uniteforpa.com/slateofeight">Slate of Eight</a></ul>
       <ul>- Ed Gainey</ul>
       <ul>- Jehosha Wright, Hilary Bendik, or Demetrius Baldwin for Magistrate</ul>
-    </p>
+    <p>
+      TO GET STARTED: fill out the form below, submit, and then click the "Continue to Step 2" button at the bottom of the page.
+    </p><p></p>
   </header>
 
-  <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSdyIUHPLzAfrkkI3kMBrdvCPhJq-zZN1BjhEcBd0wGzNkEKLw/viewform?embedded=true" width="100%" height="2100" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
-  </br>
+  <iframe src="https://docs.google.com/forms/d/e/1FAIpQLSdyIUHPLzAfrkkI3kMBrdvCPhJq-zZN1BjhEcBd0wGzNkEKLw/viewform?embedded=true" marginheight="0" marginwidth="0" width="100%" height="2100" frameborder="0">Loading…</iframe>
+  <br>
   <a href="/voters" class="Button" data-js="proceed-button">Continue to Step 2: Building Your List</a>
 
   <div class="floaty-button">
     <a href="/voters" class="Button Button--white" data-js="proceed-button">Continue to Step 2: Building Your List</a>
   </div>
 
-</body>
 
-</html>
\ No newline at end of file
+
+</body><style type="text/css"></style><style type="text/css"></style></html>
```

Comment:
What is this for?

Reply:
Not sure how that even got there, to be honest. Removing in the next commit.